blazor/speeddial/items.md

```diff
@@ -129,3 +129,24 @@ Below example demonstrates the Speed Dial items with applied Zoom effect.
 ## Template
 
 The Speed Dial supports to customize the action items and entire pop-up container by setting [ItemTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_ItemTemplate) and [PopupTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_PopupTemplate) tag directive. For more details about templates, check out the link [here](https://blazor.syncfusion.com/documentation/speeddial/getting-started).
+
+## HTML attribute support in speeddial items
+
+You can add HTML attributes for the Blazor Speed Dial action items by using [HtmlAttributes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SpeedDialItem.html#Syncfusion_Blazor_Buttons_SpeedDialItem_HtmlAttributes) property. You can also add them as inline attributes or by using the `@attributes` directive.
+
+Below example demonstrates the Speed Dial items with HTML attributes.
+
+```cshtml
+
+@using Syncfusion.Blazor.Buttons
+
+<SfSpeedDial Content="Edit"> 
+    <SpeedDialItems>
+        <SpeedDialItem Text="Cut" IconCss="e-icons e-cut" style="color:red;"></SpeedDialItem>
```

Comment:
Define items properly, use text and icon for all items

Reply:
I have updated it as per comments.